functionary/core/admin/socialapplication.py

```diff
@@ -0,0 +1,103 @@
+from allauth.socialaccount.models import SocialApp
```

Comment:
I guess the thing I'm asking is: if we dropped the entire ui app and did a completely different implementation of the UI outside of django, would this SocialApp stuff still be something you want / need to configure via the admin pages?  Or would it also go away?

Reply:
I think we would still want to use it since it saves us from having to decode the tokens and parsing the user data fields into the right spot in the object.